.github/issue_template.md

```diff
@@ -0,0 +1,14 @@
+## Persona (link)
```

Comment:
with Issue template I think you can have different types by adding a header to it. For example we could add separate template just for bug reports and one for Feature Request/UX improvements

```
---
name: Bug Report 
about: Describes a bug found in the KF Data Tracker UI

---

### UI Version

Provide the specific semantic version number of the UI (example: 1.7.0, 1.6.5, etc). This can be found in the package.json

### URL 

Provide the specific page url where the issue was noticed 


### ScreenShot(s)

Provide a screenshot(s) to quickly evaluate the visual aspect of the issue 


### Steps To Reproduce

Provide the series of steps/interactions taken to produce the issue, this greatly helps expedite development efforts in reproducing the bug. 
1. clicked "button"
2. input text "foo bar" in the search box
3. clicked "submit"
4. screen borked 😢 

### Expected Behavior 

What behavior did you expect to happed after following the Steps To Reproduce. 


### Observed Behavior 

What behavior happened that caused you to report the issue?


```